openedx/core/djangoapps/programs/tasks.py

```diff
@@ -551,21 +552,21 @@ def award_course_certificate(self, username, course_run_key):
         )
         return
 
-    visible_date = available_date_for_certificate(course_overview, certificate)
+    available_date = available_date_for_certificate(course_overview, certificate)
     LOGGER.info(
         f"Task award_course_certificate will award a course certificate to user {user.id} in course run "
-        f"{course_key} with a visible date of {visible_date}"
+        f"{course_key} with an available date of {available_date}"
     )
```

Comment:
Is there any reason we need to calculate the `available_date` here or log it? If this is all handled by Credentials now, this seems unnecessary now?

Reply:
good catch, back to you